packages/dashboard/src/customization/widgets/rectangleSymbol/icon.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
curious: is it better to just save this as an svg file and import it ?

Reply:
i don't see how we would do that because the svg depends upon the props to be rendered